src/ts/pklLspDistribution.ts

```diff
@@ -66,10 +66,10 @@ export const bundledDistribution: LspDistribution = {
 const getLspVersion = async (jarPath: string): Promise<Semver | undefined> => {
   const javaDistribution = await getJavaDistribution();
   const { stdout } = await execFile(javaDistribution.path, ["-jar", jarPath, "--version"]);
-  const versionStr = stdout
+  const stdoutParts = stdout
     .replace(/\r?\n$/, "")
-    .split(" version ")
-    .findLast((it) => true);
+    .split(" version ");
+  const versionStr = stdoutParts[stdoutParts.length - 1];
```

Comment:
`findLast()` is causing TypeScript to error (this is a new-ish APi finalized in 2023).

We can use it, but I'd need to bump the TypeScript dependency in this same PR, and it seems more sensible to limit the blast radius here.